cloudinit/config/cc_ansible.py

```diff
@@ -141,8 +142,11 @@ def install(self, pkg_name: str):
                 "-m",
                 "pip",
                 "install",
-                "--break-system-packages",
             ]
+            if os.path.join(
+                sysconfig.get_path("stdlib"), "EXTERNALLY-MANAGED"
+            ):
```

Comment:
I think you meant the following:
```suggestion
            if os.path.exists(
                os.path.join(
                    sysconfig.get_path("stdlib"), "EXTERNALLY-MANAGED"
                )
            ):
```

Reply:
Yes, thanks!